vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -116,7 +120,7 @@ module BasicFunctions =
     let sampleFunction2 (x:int) = 2*x*x - x/5 + 3
 
     let result2 = sampleFunction2 (7 + 4)
-    printfn "The result of applying the 1st sample function to (7 + 4) is %d" result2
```

Comment:
perhaps:   "The result of applying sampleFunction2 to 

and ditto for other examples ...